src/diffusers/models/transformers/transformer_qwenimage.py

```diff
@@ -197,21 +229,42 @@ def rope_params(self, index, dim, theta=10000):
     def forward(
         self,
         video_fhw: Union[Tuple[int, int, int], List[Tuple[int, int, int]]],
-        txt_seq_lens: List[int],
-        device: torch.device,
+        txt_seq_len: Optional[Union[int, torch.Tensor]] = None,
+        device: torch.device = None,
+        txt_seq_lens: Optional[List[int]] = None,
```

Comment:
```suggestion
        txt_seq_lens: Optional[List[int]] = None,
        device: torch.device = None,
        txt_seq_len: Optional[Union[int, torch.Tensor]] = None,
```

I moved the new argment to the last otherwise it would silently break if people call this function without using the positional argument. e.g. 
 with this code, the second input would now become text_seq_len, even though it was `text_seq_lens`
```py
pos_emb(image_shapes, [128, 64], device)
```